README.md

```diff
@@ -143,6 +144,20 @@ component.
 #### Inheritance
 
 The properties of the [`Box`](#box) component are also available.
+### Avatar
+
+[View story source](https://github.com/balena-io-modules/rendition/blob/master/src/components/Avatar/story.js)
+
+#### Props
+
+| Name   | Type   | Default   | Required   | Description   |
+| ------ | ------ | --------- | ---------- | ------------- |
+| `email`| `string`| - | - | The email to fetch the gravatar |
+| `fullName`| `string`| - | - | Full name will be splitted and will grab only the first letters  |
+| `src`  | `string`| - | - | The path to an image that should be displayed in the background |
+| `size`  | `string` | `medium` | - | The size of the avatar (values: 'xsmall' 'small' 'medium' 'large' 'xlarge' or css size )|
```

Comment:
Normally we don't use Grommet's sizes, as it is too flexible. We probably want 2 sizes, probably normal and `emphasized` (how we do it for the spinner, input, etc.)